accessman.go

```diff
@@ -346,6 +507,10 @@ func (a *accessMan) checkIncomingConnBanScore(remotePub *btcec.PublicKey) (
 func (a *accessMan) addPeerAccess(remotePub *btcec.PublicKey,
 	access peerAccessStatus) {
 
+	acsmLog.DebugS(ctxb, "Adding peer access",
+		"peer",
+		btclog.Hex("pubkey", remotePub.SerializeCompressed()), "access", access)
```

Comment:
I think we should have a helper for public keys. Something like `lnutils.PubKey()` that does the `btclog.Hex(name, pubKey.SerializeCompressed())` to shorten this.

Reply:
Went with elle's suggestion to make a shared context. So this way we only ever have one